### ISSUE TITLE
Add a New Device

### ISSUE THREAD

**Issue by gongyj0948** [cryptoadvance/specter-desktop#489]
When click Get over USB, Respond <unexpected token < in JASON at position 0.

**Comment by ben-kaufman** [comment]
Could you please provide more details on that? Like your OS, Specter version used, how do you use Specter (ie. the desktop app/ with mynode/ from pip/ something else...)

**Comment by gongyj0948** [comment]
Never mind, I figured that it's likely, not 100% sure, due to my router refuses to connect. When I use specter Desctop, HWI bridge (URL http://127.0.0.1:25441/hwi/api/) to connect with myNode, then when I try to connect with WH wallet, the problem shows up.

Then I deleted the Specter Desktop, only use myNode Specter App, I figured that 127.0.0.1 refuse to connect. So I sure it not Specter's problem. 

Thank you for asking!
